core/network/impl/peer_view.cpp

```diff
@@ -109,6 +109,16 @@ namespace kagome::network {
     return remote_view_.sharedAccess([](const auto &rv) { return rv.size(); });
   }
 
+  void PeerView::printStoragesLoad() const {
+    const size_t remote_veiw_size =
+        remote_view_.sharedAccess([](const auto &rv) { return rv.size(); });
+
+    SL_TRACE(logger,
+             "[Peer View storages statistics]:"
+             "\n\t-> remote_view={}",
+             remote_veiw_size);
```

Comment:
Can we use single-line messages, at least over several messages? Messages with line-breaks can spoil log or break works some log-readers